src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java

```diff
@@ -129,6 +131,35 @@ public void mapEnvelope_sets_envelope_id_to_null_when_jurisdiction_does_not_supp
         assertThat(exceptionRecord.envelopeId).isNull();
     }
 
+    @Test
+    public void mapEnvelope_sets_awaitingDcnProcessing_to_no_when_envelope_contains_payments() {
```

Comment:
doesn't match with assertion .You mean `mapEnvelope_sets_awaitingDcnProcessing_to_yes_when_envelope_contains_payments` ?

Reply:
renamed the test